src/steamship/agents/examples/my_assistant.py

```diff
@@ -26,18 +27,20 @@ def __init__(self, **kwargs):
                 SearchTool(),
                 DalleTool(),
             ],
-            llm=OpenAI(self.client),
+            llm=OpenAI(self.client, temperature=0),
+            conversation_memory=MessageWindowMemory(k=2),
         )
 
     @post("prompt")
-    def prompt(self, prompt: str) -> str:
+    def prompt(self, prompt: str, context_id: Optional[uuid.UUID] = None) -> str:
```

Comment:
I haven't yet read ahead, but in general I continue to really like your way of thinking about context